.devcontainer/teaching_vacancies_bashrc.sh

```diff
@@ -8,3 +8,81 @@ PATH=$PWD/bin:/workspace/bin:/usr/local/bundle/bin:$PATH
 
 ## Enable Git bash completion
 source /usr/share/bash-completion/completions/git
+
+## Aliases
```

Comment:
I thought was worth it to have some common quick aliases for test execution and git. Feel free to disagree/propose other aliases.